src/migration/1669932953007-rpcs.ts

```diff
@@ -0,0 +1,158 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class rpcs1669932953007 implements MigrationInterface {
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+        CREATE SCHEMA http; CREATE EXTENSION http WITH SCHEMA http;
+
+        create or replace function iasql_engine_health() returns text
+        language plpgsql security definer
+        as $$
+        declare
+          _content text;
+        begin
+          select content into _content from http.http_get('http://localhost:8088/health');
+          return _content;
+        end;
+        $$;
+
+        create or replace function iasql_version() returns text
+        language plpgsql security definer
+        as $$
+        declare
+          _content text;
+        begin
+          select content into _content from http.http_get('http://localhost:8088/v1/version');
```

Comment:
Can we use a variable for the `localhost:8088`? I've already pointed an issue on Docker Desktop for Mac (same in Windows) in which the `network_mode: host` doesn't work. In order to make that work, these `localhost:8088`'s need to be changed to `host.docker.internal:8088`.

Reply:
That *should* be doable with a new `config` property. Do you already have a PR with that change you're mentioning? I can rebase on top of that and use that property.

Reply:
No I don't have it as a PR, but I can add it tomorrow. We might do it as a follow-up since I think this should be merged soon?

Reply:
@mtp1376 I think I have addressed this, and also made changes to the migration. We'll see if it works in a bit!